spec/_widget/components/search.spec.js

```diff
@@ -87,8 +87,8 @@ describe('Search', () => {
         'Account Activation': 5
       },
       'enable dnssec': {
-        'DNSSEC': 1,
-        'Why DNSSEC and Secondary DNS may not work together': 3
+        'DNSSEC': 2,
```

Comment:
@dperrymorrow This is the minimum rank for this article/search term (not # of articles). I've kept them relatively tight so that we can notice larger changes. These are not an exhaustive list of results for the term, but the articles that we've said MUST show up for the search term.

If you run the specs, you'll see warnings for search terms where we can tighten up the results (if there are any).

I recommend we add `Enabling DNSSEC` here as a `1` as well.